public/scripts/color-scheme.js

```diff
@@ -9,9 +9,9 @@ const DEFAULT_COLOR_SCHEME = COLOR_SCHEME_SYSTEM;
 const CLASS_DARK_MODE = 'dark';
 const CLASS_LIGHT_MODE = 'light';
 
-let activeColorScheme = DEFAULT_COLOR_SCHEME;
-
 export default function initColorScheme(document, storage) {
+  let activeColorScheme = null;
```

Comment:
Should this be `null` or `DEFAULT_COLOR_SCHEME`. The latter has the main advantage that nothing will be stored in local storage until the user has actually chosen a color scheme.

Reply:
This has been set to `DEFAULT_COLOR_SCHEME` as per the advantage above. Additionally, the `DEFAULT_COLOR_SCHEME` is no longer explicitly loaded [later in the file](https://github.com/simple-icons/simple-icons-website/blob/6bcf096af11224ab73b36358130e76575266b2ac/public/scripts/color-scheme.js#L28) either, for the same reason.